oscore-cf/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.californium</groupId>
+    <artifactId>parent</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>oscore-cf</artifactId>
+  <dependencies>
+	<dependency>
+		<groupId>com.augustcellars.cose</groupId>
+		<artifactId>cose-java</artifactId>
+		<version>0.9.7</version>
+	</dependency>
+	<dependency>
+		<groupId>org.eclipse.californium</groupId>
+		<artifactId>scandium</artifactId>
+		<version>2.0.0-SNAPSHOT</version>
+		</dependency>
+  	<dependency>
+	 	<groupId>com.upokecenter</groupId>
+  		<artifactId>cbor</artifactId>
+  		<version>2.4.1</version>
```

Comment:
this version seems to be more than 2 years old. Any chance of using a more (or better: the most) recent one?

Reply:
I will try out the new one.